otterdog/eclipse-cyclonedds.jsonnet

```diff
@@ -71,6 +71,17 @@ orgs.newOrg('eclipse-cyclonedds') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('cyclonedds-insight') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      default_branch: "master",
```

Comment:
for new repos it is advised to follow the new standard naming scheme of main, when you remove this setting, it will inherit the default value.

Reply:
I think it is best to follow the same naming in all the project repositories. Otherwise it becomes a bit of mess, IMO. For new projects, certainly ...